grafana-plugin/.eslintrc.js

```diff
@@ -8,6 +8,10 @@ module.exports = {
     'import/internal-regex':
       '^assets|^components|^containers|^contexts|^icons|^models|^network|^pages|^services|^state|^utils|^plugin',
   },
+  parserOptions: {
```

Comment:
is it needed here since we have it in overrides already? Seems it's a duplication in this case?

Reply:
I'd say not as I undo-ed the other packages' updates. But I believe once you bump grafana/eslint, it is needed.